tests/pkg/littlefs/Makefile

```diff
@@ -4,4 +4,7 @@ USEMODULE += littlefs
 USEMODULE += embunit
 USEMODULE += mtd_emulated
 
+# silence expected errors
+CFLAGS += -DLOG_LEVEL=LOG_NONE
```

Comment:
can you elaborate on what those are? wouldn't another log level lower than `LOG_DEBUG` be enough?

Reply:
There is a test that erases the simulated MTD, tries to mount the non-existent filesystem, checks for the failure, formats the MTD, then tries to mount again, and finally checks for success.

This sequences causes errors (during the failed mount), to be mixed into the test's output to stdout. If the CI run can handle this we can leave it on, but I assume that it would mess up attempts to parse the test's output.

Reply:
Here is what the output looks like with logging silenced:
```
RIOT native interrupts/signals initialized.
RIOT native board initialized.
RIOT native hardware initialization complete.

Help: Press s to start test, r to print it is ready
READY
s
START
........
OK (8 tests)
```

Here it is with errors turned on:
```
RIOT native interrupts/signals initialized.
RIOT native board initialized.
RIOT native hardware initialization complete.

Help: Press s to start test, r to print it is ready
READY
s
START
main(): This is RIOT! (Version: 2024.10-devel-264-g259f2-littlefs-logging)
.lfs: Corrupted dir pair at {0x0, 0x1}
lfs: Corrupted dir pair at {0x0, 0x1}
.......
OK (8 tests)
{ "threads": [{ "name": "idle", "stack_size": 8192, "stack_used": 436 }]}
{ "threads": [{ "name": "main", "stack_size": 12288, "stack_used": 2904 }]}
```

Reply:
You should be able to try with `make test` in the test folder yourself. AFAIK the python test script only checks for "OK" in the output.

Reply:
I just tried it. You are correct. The log messages are harmless. Shall I turn them back on then?

Reply:
Thanks for testing it!

But I think we can leave as is, since the old version of the littlefs Makefile actually turned all logs off by default as well, so I guess silencing expected errors in the test is okay :)